Library/Formula/midnight-commander.rb

```diff
@@ -28,4 +28,10 @@ def install
                           "--enable-vfs-sftp"
     system "make install"
   end
+
+  test do
+    output = `#{bin}/mc -V`
```

Comment:
Use `shell_output` instead of backticks which will also do the `assert_equal` below for you.